src/Aquarius.ONE.ClientSDK/Enterprise/Core/CoreApi.cs

```diff
@@ -460,10 +460,15 @@ public async Task<List<ProductOffering>> GetProductOfferingsAsync(EnumProductOff
                     expand = "?expand=tenant,feature";
                     break;
             }
+
+            var tenantIdQueryParameter = string.IsNullOrEmpty(tenantId)
+                ? ""
+                : $"{(string.IsNullOrEmpty(expand) ? "?" : "&")}tenantId={tenantId}";

```

Comment:
There is probably a .NET library to compose the query string for you, but this is fine.